### PR TITLE
Make util method more generic, update help

### PR DESCRIPTION
**Pull request by djyou**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py [hunk 1]
```diff
@@ -441,12 +441,11 @@
     examples:
         - name: Import from the same registry.
           text: >
-            az acr image import -n registry --source-image registry.azurecr.io/repository:tag -t repo:tag .
+            az acr image import -n registry --source-image registry.azurecr.io/repository:tag -t targetRepository:targetTag
```
Comment (djyou) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py]: Still not sure about this string `targetRepository:targetTag` ... Basically repository doesn't allow upper case. Let me know what you think. /cc @NatalieGong

Comment (NatalieGong) [reply src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py]: Oh ok. I can turn it into lower case if that is the case.

Comment (djyou) [reply src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py]: I can make it lower case if you prefer.

Comment (ehotinger) [reply src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py]: definitely don't put non-working examples in the examples area 👍 / or just use `repoistory:tag`

**Review by NatalieGong — APPROVED** [review]

**Pull request merged**